install/src/command.rs

```diff
@@ -229,12 +229,13 @@ fn new_update_manifest(
         let recent_blockhash = rpc_client.get_latest_blockhash()?;
 
         let lamports = rpc_client
-            .get_minimum_balance_for_rent_exemption(SignedUpdateManifest::max_space() as usize)?;
+            .get_minimum_balance_for_rent_exemption(SignedUpdateManifest::MAX_SPACE as usize)?;
```

Comment:
Only today did I learn about this whole process of deploy update manifests :sweat_smile: 

Looks like it's only used with the net scripts